platform-integrations/claude/plugins/evolve-lite/README.md

```
@@ -55,6 +55,88 @@ You can also manually invoke `/evolve-lite:learn` at any time.
+```
+/evolve-lite:sync
+> Synced 2 repo(s): alice (+2 added, 0 updated, 0 removed), bob (+0 added, 1 updated, 0 removed)
+```
+
+If `sync.on_session_start: true` is set in config, this runs automatically at the start of each session.
+
+### Unsubscribing
+
+Use `/evolve-lite:unsubscribe` to remove a subscription and delete its locally cloned files:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)